project/CONTRIBUTING.md

```diff
@@ -180,9 +180,9 @@ namespace Foo;
 
 interface BarInterface
 {
-
-  // NEXT_MAJOR: Uncomment this method
   /**
+   * NEXT_MAJOR: Uncomment this method
```

Comment:
Ok but this will be done as a PR against master/major branch and in this case we will review this